src/package_io/constants.star

```diff
@@ -31,6 +31,8 @@ GENESIS_VALIDATORS_ROOT_PLACEHOLDER = "GENESIS_VALIDATORS_ROOT_PLACEHOLDER"
 
 DEFAULT_SNOOPER_IMAGE = "ethpandaops/json-rpc-snoop:1.1.0"
 
+DEFAULT_ETHEREUM_METRICS_EXPORTER_IMAGE = "ethpandaops/ethereum-metrics-exporter:0.22.0"
```

Comment:
You might want to use latest here instead of a tagged version. Just so we don't need to make a new PR for each time this image is updated. We have a static version set for json rpc snoop as that image is not planned to change that frequently.

Reply:
I don't think we update EME enough for this to be an issue (would also really prefer to not use latest in a testing tool like this)